Jenkinsfile

```diff
@@ -1,33 +1,33 @@
 @Library('global-jenkins-library@2.3.1') _
 
 buildInfo = getBuildInfo()
+dockerIoVisibility = Registries.EXTERNAL_DOCKERIO_HOST
```

Comment:
Not sure why we declare a variable for an enum member but not for the other :thinking:

Reply:
The first one is here to remove duplication :)

Reply:
I don't think having duplicates on enum members/constants is an issue. Do you think so?

Reply:
It expresses here a same policy applies for different stages. Not sure why we are stuck here :thinking: :slightly_smiling_face:

Reply:
IMHO, using the `Registries.EXTERNAL_DOCKERIO_HOST` would express the same policy. I find it clearer to have a 'symmetrical' pattern: whether we use new variables for all visibilities, whether we don't use variables,
But you're right, we shouldn't be stuck there. That's very minor, just a personal preference. Let's keep moving!